eng/RuntimeIdentifier.props

```diff
@@ -45,6 +45,12 @@
     <TargetRid Condition="'$(TargetRid)' == ''">$(_targetRidOS)-$(TargetArchitecture)</TargetRid>
   </PropertyGroup>
 
+  <PropertyGroup Label="CalculateTargetAuxiliary">
+    <!-- Will include wasm64 once we support it -->
+    <TargetsWasm Condition="'$(TargetArchitecture)' == 'wasm'">true</TargetsWasm>
+    <TargetsSingleThreadedWasm Condition="'$(TargetsWasm)' == 'true' and '$(WasmEnableThreads)' != 'true'">true</TargetsSingleThreadedWasm>
```

Comment:
We now have `TargetsWasm` in `eng/OSArch.props`, we should move `TargetsSingleThreadedWasm` there as well.

Reply:
Done